test/spec/modules/sonobiBidAdapter_spec.js

```diff
@@ -539,39 +539,6 @@ describe('SonobiBidAdapter', function () {
       ]);
     });
 
-    it('should return a properly formatted request with userid as a JSON-encoded set of User ID results', function () {
```

Comment:
can you add a test that says basically, it doesnt choke if a publisher specifies this, even though it doesnt do anything

Reply:
Ya we can do that. I restored the test and removed the check on line 550. 

The other 2 checks for data.ref and data.s should stay the same. These are additional checks for other values the buildRequest function sets but should not be effected by the presence of a userid.